src/cli/commands/workspaces.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import type Config from '../../config.js';
+import {MessageError} from '../../errors.js';
+import type {Reporter} from '../../reporters/index.js';
+
+const invariant = require('invariant');
+
+export function setFlags(commander: Object) {}
+
+export function hasWrapper(commander: Object, args: Array<string>): boolean {
```

Comment:
Do we really need to define this function? I thought it defaulted to `true`.

Reply:
I think @voxsim worked to standardize the commands a few months ago, and made this required in practice if not in theory. I'm fine with it, it feels less magic (even if the whole wrapper concept is a bit opaque).